sdk/keyvault/keyvault-keys/review/keyvault-keys.review.api.md

```diff
@@ -0,0 +1,295 @@
+## API Report File for "@azure/keyvault-keys"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import * as msRest from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface CreateEcKeyOptions extends CreateKeyOptions {
+    curve?: JsonWebKeyCurveName;
+    hsm?: boolean;
+}
+
+// @public
+export interface CreateKeyOptions {
+    enabled?: boolean;
+    expires?: Date;
+    keyOps?: JsonWebKeyOperation[];
+    // (undocumented)
+    keySize?: number;
+    notBefore?: Date;
+    requestOptions?: msRest.RequestOptionsBase;
```

Comment:
any reason that this type shouldn't just be `extends RequestOptions`, which is used frequently below?